examples/cronpolicy/admission-webhook-without-high-availability.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: checkpoint.devsisters.com/v1
+kind: CronPolicy
+metadata:
+  name: admision-webhook-without-high-availability
+spec:
+  suspend: false
+  schedule: "* * * * *"
+  resources:
+  - group: admissionregistration.k8s.io  
```

Comment:
Currently it receives all GVK strings to match resources, but matching only with `Kind`(which Kyverno does) will be very handy because `version` can change among various kubernetes versions.

Reply:
This would be impossible because `kube-rs`'s dynamic client takes all GVK strings. If we want to take only `Kind`, we have to query `api-resources` to get the actual GVK.